### ISSUE TITLE
Nice! How was your experience with it so far?

### ISSUE THREAD

**Issue by fullofcaffeine** [Justinfront/YouTube-tutorials-haxeJava#1]
Hi Justin,

I'm glad I found this repo. Interesting stuff. How was your experience translating the content of the tutorials to hxJava so far? What are the major bumps you've been through? Any major blocks as of the latest Haxe/hxJava versions?

Cheers!

**Comment by Justinfront** [comment]
Hi fullofcaffeine
It was a long time back, before last conference.  I think some of the more complex tutorials get into usual java inheritance complexity and so are too much effort and not really ideal from haxe viewpoint.  Also I can't remember if they needed some lib-java etc... that were not compatible with that version of haxe. I guess should look through and work out which ones are worth finishing if you have suggestions? If your into Java haxe there are probably a few things that I can point you towards, but I guess most of them are in my github or gists.  One aspect I had was in my Letters repository, I got the js version working but struggled with the Java version, perhaps a haxe version so I should look again.  The other thing I found with Java is only some libraries you could generate externals for, I think I should maybe have another look as I am sure Caue and others have made huge strides with java target.
The latest stuff I have done with Haxe Java is probably run the finda which needs a few changes maybe not updated, and hxDaedalus is definitely worth a look it abstracts some basic drawing accross a range of haxe targets and is really at extension of Letters with obviously the other contributors input.

**Comment by Justinfront** [comment]
did you look at https://github.com/Justinfront/wwx2014_justinfront it has some non swing stuff.  I think really the next project for Java is probably to get lwjgl as a snow target ....!  But I am not sure I have the skill and motivation, but I would think it would be very feasible and might be very useful. Java compile is likely faster than c++.

**Comment by fullofcaffeine** [comment]
Thanks, will look into the repo! 

I'm just starting and considering Haxe as a nicer language to use the Java eco-system, specially for Desktop apps that need to be close to the native performance while still being cross-platform. I'm between Haxe and jRuby. 

Right now, I tend to the Ruby side, since that's the language I use for most of my projects, but I'd love to try Haxe since I personally think it's more interesting in some ways and also, in theory, could allow for more code reuse between targets and platforms.

Will let you know how it goes.

**Comment by fullofcaffeine** [comment]
Btw, Are you going to wwx2015? If so, I'll meet you there!

**Comment by Justinfront** [comment]
Yep but not yet sorted transport and accomodation prob should short it today :)

**Comment by Justinfront** [comment]
By the way quite a few experimental projects I have tried with java target as well as targetting other stuff.
https://github.com/hxDaedalus/hxDaedalus-swing-examples/blob/master/hxDaedalus-Examples/src/BitmapPathfinding04.hx
https://github.com/Justinfront/Jigsawx/blob/master/jigsawxtargets/hxjava/JigsawxJava.hx
https://github.com/Justinfront/crossAWD2/blob/master/src/targets/java/AwdJava.hx
Letters I should finish it had bugs and ended up concentrating on js target.
https://github.com/Justinfront/hx_swing_examples/blob/master/src/hxswing/TwoWindowsExample.hx
https://github.com/Justinfront/finda/blob/master/src/finda/Main.hx
Then there is that tutorial I posted on old.haxe.org which I have the same version here in different format.
https://docs.google.com/document/d/1tKlZ4Qh4KXT-WkDDlhIzjxgTk0WVMg72_J6rdLrsLy0/edit#heading=h.swuk14xubtah

**Comment by Justinfront** [comment]
Somewhere I have a java rich textfield not hard to setup but useful not sure if I uploaded it anywhere or where it is.
